packages/sdk-webos/src/deviceManager.ts

```diff
@@ -43,6 +43,9 @@ export const launchWebOSimulator = async (target: string | boolean) => {
     const availableSimulatorVersions = getDirectories(path.join(webosSdkPath, 'Simulator'));
 
     if (target === true) {
+        if(availableSimulatorVersions.length == 0){
```

Comment:
use equality with type checking `availableSimulatorVersions.length === 0` also run linter for this file because I see there a missing spaces and indentation

Reply:
Fixed